### PR TITLE
WA Groups: Updated receiving messages with new schema changes

### PR DESCRIPTION
**Pull request by madclaws**
## Summary
 Target issue is #3386

### CODE REVIEW

File: lib/glific/communications/message.ex [hunk 1]
```diff
@@ -5,6 +5,9 @@ defmodule Glific.Communications.Message do
   import Ecto.Query
   require Logger
 
+  alias Glific.WAGroup.WAMessage
```
Comment (AkhileshNegi) [inline lib/glific/communications/message.ex]: alias these two inside Glific.{}

File: lib/glific/communications/message.ex [hunk 2]
```diff
@@ -206,17 +211,10 @@ defmodule Glific.Communications.Message do
   end
 
   @spec do_receive_message(Contact.t(), map(), atom()) :: :ok | {:error, String.t()}
-  defp do_receive_message(contact, %{organization_id: organization_id} = message_params, type) do
+  defp do_receive_message(contact, %{organization_id: _organization_id} = message_params, type) do
```
Comment (AkhileshNegi) [inline lib/glific/communications/message.ex]: if we are not using %{organization_id: _organization_id} then just use `message_params`

```
  defp do_receive_message(contact, message_params, type) do
  ```

File: test/glific_web/providers/maytapi/controllers/message_controller_test.exs [hunk 3]
```diff
@@ -307,26 +294,19 @@ defmodule GlificWeb.Providers.Maytapi.Controllers.MessageControllerTest do
...
       # ensure the message has been received by the mock
-      assert_receive :received_message_to_process
-
-      assert message.sender.last_message_at != nil
-      assert true == Glific.in_past_time(message.sender.last_message_at, :seconds, 10)
-
-      # Sender should be stored into the db
-      assert message.sender.phone ==
-               get_in(@text_message_webhook_new_group, ["user", "phone"])
+      # assert_receive :received_message_to_process
```
Comment (AkhileshNegi) [inline test/glific_web/providers/maytapi/controllers/message_controller_test.exs]: do we need this?

File: test/glific_web/providers/maytapi/controllers/message_controller_test.exs [hunk 4]
```diff
@@ -366,26 +346,19 @@ defmodule GlificWeb.Providers.Maytapi.Controllers.MessageControllerTest do
...
       # ensure the message has been received by the mock
-      assert_receive :received_message_to_process
-
-      assert message.sender.last_message_at != nil
-      assert true == Glific.in_past_time(message.sender.last_message_at, :seconds, 10)
-
-      # Sender should be stored into the db
-      assert message.sender.phone ==
-               get_in(@media_message_webhook, ["user", "phone"])
+      # assert_receive :received_message_to_process
```
Comment (AkhileshNegi) [inline test/glific_web/providers/maytapi/controllers/message_controller_test.exs]: do we need this?

File: test/glific_web/providers/maytapi/controllers/message_controller_test.exs [hunk 5]
```diff
@@ -406,26 +379,19 @@ defmodule GlificWeb.Providers.Maytapi.Controllers.MessageControllerTest do
...
       # ensure the message has been received by the mock
-      assert_receive :received_message_to_process
-
-      assert message.sender.last_message_at != nil
-      assert true == Glific.in_past_time(message.sender.last_message_at, :seconds, 10)
-
-      # Sender should be stored into the db
-      assert message.sender.phone ==
-               get_in(media_message_payload, ["user", "phone"])
+      # assert_receive :received_message_to_process
```
Comment (AkhileshNegi) [inline test/glific_web/providers/maytapi/controllers/message_controller_test.exs]: do we need this?

**Review by AkhileshNegi — CHANGES_REQUESTED** [review]

File: test/glific_web/providers/maytapi/controllers/message_controller_test.exs (hunk 3 above)
Comment (madclaws) [reply test/glific_web/providers/maytapi/controllers/message_controller_test.exs:309]: This is a test where we create a contact when we receive_message via gupshup and then again receive message via maytapi. So above lines are gupshup related

File: test/glific_web/providers/maytapi/controllers/message_controller_test.exs (hunk 4 above)
Comment (madclaws) [reply test/glific_web/providers/maytapi/controllers/message_controller_test.exs:361]: This is a test where we create a contact when we receive_message via gupshup and then again receive message via maytapi. So above lines are gupshup related

**Review by AkhileshNegi — APPROVED** [review]

**Pull request merged**